sass/utils/_variables.scss

```diff
@@ -16,6 +16,7 @@ $transparent: rgba(0, 0, 0, 0);
 
 // Page Variables
 $border-gray: 1px solid #ddd;
+$announcement-bar-height: 30px;
```

Comment:
Was 22px, seems to need to be 30px. @jwilander, specifically adding you to the review because I want to verity the assumption that this element is supposed to be displayed as a block, and that we can rely on a fixed height as such.